docs/source/examples/vision/checks/distribution/heatmap_comparison.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>I would write at the beginning that "The heatmap comparison is a kind of a data drift check"</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/901/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>